library/src/main/java/com/datatorrent/lib/io/fs/AbstractSingleFileOutputOperator.java

```diff
@@ -52,6 +81,7 @@ protected String getFileName(INPUT tuple)
   public void setOutputFileName(String outputFileName)
   {
     this.outputFileName = outputFileName;
+    partitionedFileName = String.format(partitionedFileNameformat, outputFileName, physicalPartitionId);
```

Comment:
Since partitionedFileName is assigned in setup() there is no need to do it here; in fact, since these setters are typically called before the operator is serialized, physicalPartitionId will always be 0 here.